packages/manager/apps/dedicated/package.json

```diff
@@ -119,7 +119,7 @@
     "oclazyload": "^1.1.0",
     "office-ui-fabric-core": "^11.0.0",
     "ovh-angular-responsive-tabs": "^4.0.0",
-    "ovh-api-services": "^9.30.0",
+    "ovh-api-services": "^9.32.0",
```

Comment:
Could have been nice to `yarn upgrade-interactive` separately on `develop` branch

Reply:
IMO, only `exchange` require a bump of `ovh-api-services` (and therefore `web`and `dedicated` manager). We don't have to bump `ovh-api-services` right now in other packages as they don't need the features introduced in `ovh-api-services@9.32.0`.

Reply:
Updated for all applications using `yarn upgrade-interactive  --latest`